jsonist.js

```diff
@@ -108,3 +108,4 @@ function isRedirect (request, response) {
 module.exports.get  = makeMethod('GET'  , false)
 module.exports.post = makeMethod('POST' , true)
 module.exports.put  = makeMethod('PUT'  , true)
+module.exports.delete  = makeMethod('DELETE'  , false)
```

Comment:
note the pattern here, the `=` are all lining up regardless of the length of the name and the comma and booleans are all lining up too, could you try and reformat these 4 lines so they line up again? `post` was previously the longest so set the padding but now `delete` is longer so will determine that.